modules/users/client/controllers/password.client.controller.js

```diff
@@ -55,13 +53,13 @@
     function onRequestPasswordResetSuccess(response) {
       // Show user success message and clear form
       vm.credentials = null;
-      vm.success = response.message;
+      Notification.success({ message: response.message, title: 'Password request successful!' });
```

Comment:
![image](https://cloud.githubusercontent.com/assets/19917701/19058411/12725a78-89a5-11e6-8512-67268773ace2.png)

@mleanos Did you mean a title like this? I thought the title and the message were decoupled as the email was not attempted to be sent, and so kept the title generic. But if this image makes more sense let me know and I can update the text.